server/logging/logging.go

```diff
@@ -3,78 +3,29 @@ package logging
 import (
 	"context"
 	"errors"
-	"io"
+
+	"github.com/woodpecker-ci/woodpecker/server/model"
 )
 
+// TODO(#742): write adapter for external pubsub provider
+
```

Comment:
```suggestion
```

This is not needed here. That's what the pubsub interface and code is for already.